packages/forms/resources/js/components/textarea.js

```diff
@@ -7,7 +7,7 @@ export default (Alpine) => {
         render: function () {
             if (this.$el.scrollHeight > 0) {
                 this.$el.style.height = '150px'
-                this.$el.style.height = this.$el.scrollHeight + 'px'
+                this.$el.style.height = this.$el.scrollHeight + 2 + 'px'
```

Comment:
```suggestion
                this.$el.style.height = (this.$el.scrollHeight + 2) + 'px'
```

Would like to avoid all weird JS `number + string` bugs if possible.